packages/components/src/components/notification/_mixins.scss

```diff
@@ -20,6 +20,17 @@
   }
 }
 
+/// @access private
```

Comment:
Is this change here intentional...?

Reply:
That seems to be an artifact of extracting the changes I wanted ... I'll sort that out